Code/Tools/ProjectManager/Source/FormLineEditWidget.h

```diff
@@ -43,12 +53,21 @@ namespace O3DE::ProjectManager
 
         virtual void setText(const QString& text);
 
+        void SetValidationState(ValidationState validationState);
+
     protected:
         QLabel* m_errorLabel = nullptr;
         QFrame* m_frame = nullptr;
         QHBoxLayout* m_frameLayout = nullptr;
         AzQtComponents::StyledLineEdit* m_lineEdit = nullptr;
 
+        //Validation icons
+        QMovie* m_processingSpinnerMovie = nullptr;
+        QLabel* m_processingSpinner = nullptr;
+        QLabel* m_validationErrorIcon = nullptr;
+        QLabel* m_validationSuccessIcon = nullptr;
+        ValidationState m_validationState;
```

Comment:
We should probably default the validation state to NotValidating, in case it gets checked without being set